presentation/src/main/java/me/androidbox/presentation/agenda/viewmodel/AgendaViewModel.kt

```diff
@@ -40,6 +42,7 @@ class AgendaViewModel @Inject constructor(
         getAuthenticatedUser()
         fetchAgendaItems(ZonedDateTime.now())
         syncAgendaItems()
+        syncFullAgendaItems()
```

Comment:
Is this the best place to sync the FullAgenda?

Reply:
Yea